### ISSUE TITLE
Editor is not visible at certain condition

### ISSUE THREAD

**Issue by ndhik** [KillerCodeMonkey/ngx-quill#68]
Hi, thanks for creating this plugin :D

I found a problem when I want to edit a description in my component using the same component to create it (create sets initial value to null, but edit sets with data retrieved from db).

When the component is in creating mode, the editor is perfectly displayed.
But when the component is editing mode, the editor is disappear, with or without initial value.
I also tested without ngModel and formControlName directives. And when editing mode, the editor not showing.

I used both template-driven and reactive form. Neither of them work in editing mode.
**creating mode**
<img width="441" alt="screen shot 2017-10-20 at 00 03 53" src="https://user-images.githubusercontent.com/1467768/31783961-48bb5f92-b52a-11e7-9b46-fb936f38a673.png">

**editing mode**
<img width="444" alt="screen shot 2017-10-20 at 00 04 03" src="https://user-images.githubusercontent.com/1467768/31783996-688ad988-b52a-11e7-8a65-805f3ef981a8.png">

any advice regarding this issue?

**Comment by KillerCodeMonkey** [comment]
Could you show me some code, a plunkr or something like that?

Did you checked your Dom if there is something missing? E.g. your editor
node. Please also check your JS console for errors.

Am 19.10.2017 19:17 schrieb "ndhik" <notifications@github.com>:

**Comment by KillerCodeMonkey** [comment]
maybe you have some misleading ngIfs or you are adding the quill-editor component outside of the angular context... code would be helpful :)

**Comment by ndhik** [comment]
here is the comparison of both dom (while edit and create).
**create**
<img width="898" alt="screen shot 2017-10-20 at 08 04 26" src="https://user-images.githubusercontent.com/1467768/31800773-7f8a2a9c-b56d-11e7-8f58-7486668244bf.png">
this shows that the editor has children divs. I also set initial value of the ngmodel by hard code. and it shows nothing wrong by setting initial value using template driven form.

**edit**
<img width="887" alt="screen shot 2017-10-20 at 08 04 44" src="https://user-images.githubusercontent.com/1467768/31800830-d4bead94-b56d-11e7-8104-f6b4dca8eca0.png">
meanwhile, there is no ql-* divs when the form is in edit mode.

my best guess now is that the initial val which is loaded from Observable, is undefined when editor is rendered (?).

I'll get back to you very soon after I minified the project repo that mimicking the issue.

**Comment by ndhik** [comment]
the ngIfs and quill is located on the correct context.

My mistake is that I did not initiate the object which the editor is bound to. so the object returned undefined when the editor loaded its resources.

When I declared and initiated an object with blank value, the editor loaded with that blank object, the app retrieve the data from db on `ngOnInit`, and the new data replaces the blank object.

Thanks for your assistance. I'm gonna close this thread.
